Dependencies/xr/CMakeLists.txt

```diff
@@ -33,7 +33,9 @@ else()
         "Source/OpenXR/XrRegistry.h"
         "Source/OpenXR/XrSupportedExtensions.h"
         "Source/OpenXR/SceneUnderstanding.h"
-        "Source/OpenXR/SceneUnderstanding.cpp")
+        "Source/OpenXR/SceneUnderstanding.cpp"
+        "Source/OpenXR/Input.h"
```

Comment:
>Input [](http://example.com/codeflow?start=23&length=5)

nit: we might want to suffix these new files with Xr even though i think i originally failed to do so with scene understanding